CMakeLists.txt

```diff
@@ -495,7 +495,7 @@ if(NOT BUILD_HEADLESS)
             ${Boost_INCLUDE_DIRS}
             ${OPENGL_INCLUDE_DIR}
             ${GLEW_INCLUDE_DIRS}
-            ${SDL_INCLUDE_DIRS}
```

Comment:
this adds `SDL_INCLUDE_DIR` but removes `SDL_INCLUDE_DIRS`. Was the latter doing anything useful, then?

Reply:
Looks like `SDL_INCLUDE_DIRS` was empty.